lib/cache/collections.go

```diff
@@ -108,13 +110,81 @@ func setupCollections(c *Cache, watches []services.WatchKind) (map[string]collec
 				return nil, trace.BadParameter("missing parameter Presence")
 			}
 			collections[watch.Kind] = &tunnelConnection{watch: watch, Cache: c}
+		case services.KindAccessRequest:
+			if c.Presence == nil {
```

Comment:
why are we checking for `Presence` here? I would assume the check for `DynamicAccess` would be necessary